nucypher/config/base.py

```diff
@@ -398,6 +398,7 @@ def __init__(self,
                  poa: bool = None,
                  light: bool = False,
                  eth_provider_uri: str = None,
+                 payment_provider_uri: str = None,
```

Comment:
Could you clarify why this is needed? See Line 409 which uses `payment_provider`.

Reply:
yep, you're totally right - there's a difference in naming convention here, we use `eth_provider_uri` and just `payment_provider`.  I think it would be cleaner if both were `*_uri`

Reply:
Yep, it would be more consistent to use the "_uri" suffix, but therefore we should change the existing parameter used for that value on L409, and not add a new one - or else we'd have two parameters for the same thing (unless I'm misunderstanding).

I'm not sure about what the side effects are for changing the existing parameter name (cc @KPrasch ).

Reply:
I think renaming this field to `payment_provider_uri` is okay, but I think a manual test against an existing configuration is warranted.  It's worth considering the TODO that reads: `Resolve code prefixing below, possibly with the use of nested configuration fields`. Also this appears to be a duplicate parameter to the one on line 409.